yellowbrick/features/projection.py

```diff
@@ -92,18 +92,24 @@ class ProjectionVisualizer(DataVisualizer):
         the visualization as defined in other Visualizers.
     """
 
-    def __init__(self, ax=None, features=None, classes=None, color=None,
+    def __init__(self, ax=None, features=None, classes=None, colors=None,
              colormap=None, target_type="auto", projection=2, alpha=0.75,
              colorbar=True, **kwargs):
 
         # TODO: add this to resolve_colors
-        if color==None and colormap==None:
-            colormap=palettes.DEFAULT_SEQUENCE
+        if colors is None and colormap is None:
+            colormap = palettes.DEFAULT_SEQUENCE
```

Comment:
Note that I did look into moving this to `resolve_colors` -- right now if both are None, `resolve_colors` just returns the default color cycle. We do still need to do something better than this though.